com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -490,7 +490,7 @@ private PlatformInterface CreatePlatformInterface()
 #else
                     configData.platformOptionsFlags.Unwrap();
 #endif
-                if (configData.IsEncryptionKeyValid())
+                if (EOSClientCredentials.IsEncryptionKeyValid(configData.encryptionKey))
```

Comment:
This seems really close to the `IsEncryptionKeyValid` function. Can we just use that?

Reply:
It is very close, however currently `EOSConfig` does not have an `EOSClientCredentials` field member, so you cannot acquire it and call the function from the instance. This will change in the upcoming changes, but moving the implementation of checking the validity of the encryption key into the `EOSClientCredentials` class first seemed like a reasonable approach to make the changes more incremental.